Cargo.toml

```diff
@@ -31,7 +31,9 @@ tokio = { version = "1.12.0", default-features = false }
 zeroize = { version = "1.2.0", default-features = false, features = ["zeroize_derive"] }
 
 # storage
-rocksdb = { git = "https://github.com/rust-rocksdb/rust-rocksdb", rev = "86d983987e7cafce90ad8a147b0b325e6007eba6", default-features = false, features = ["lz4"], optional = true }
+# Can use upstream again once iOS support is merged
```

Comment:
Maybe add https://github.com/rust-rocksdb/rust-rocksdb/pull/637 here? Otherwise one might not know what this is about